datadog-checks-base/datadog_checks/checks/win/winpdh.py

```diff
@@ -15,18 +15,32 @@
 class WinPDHCounter(object):
     # store the dictionary of pdh counter names
     pdh_counter_dict = defaultdict(list)
+    _use_en_counter_names = False
 
     def __init__(self, class_name, counter_name, log, instance_name = None, machine_name = None, precision=None):
         self.logger = log
-        self._get_counter_dictionary()
-        class_name_index_list = WinPDHCounter.pdh_counter_dict[class_name]
-        if len(class_name_index_list) == 0:
-            self.logger.warn("Class %s was not in counter name list, attempting english counter" % class_name)
+
+        try:
+            self._get_counter_dictionary()
+            class_name_index_list = WinPDHCounter.pdh_counter_dict[class_name]
+        except WindowsError as e:
+            WinPDHCounter._use_en_counter_names = True
+            self.logger.warning("Unable to get counter translations; attempting default English names")
+            pass
```

Comment:
You can remove `pass`.

Reply:
I prefer to have the explicit `pass` to call out that we're eating (i.e. not re-raising) the exception.  Just notation for me.  If it's important (or considered bad style) I can remove it.

Reply:
Yeah, it's not very pythonic. You could compare it to putting `pass` at the end of every block or function.